app/assets/stylesheets/components/_buttons.scss

```diff
@@ -13,9 +13,19 @@
   }
 }
 
-
 .btn-secondary {
   color: white;
   border-radius: 8px;
-  width: 345px ;
+  width: 345px;
```

Comment:
@d-tello FYI I changed the button to display block instead of a specific width to be responsive. This might be one of the conflicts.